plugin.video.embycon/resources/lib/utils.py

```diff
@@ -87,7 +87,7 @@ def getPlayUrl(self, id, media_source, force_transcode, play_session_id):
             playurl = playurl % (server, id, media_source_id, play_session_id, deviceId, bitrate, playback_max_width)
             if playback_video_force_8:
                 playurl = playurl + "&MaxVideoBitDepth=8"
-            playurl = playurl + "&api_key=" + user_token
+            playurl = playurl + "&api_key=" + user_token + "|verifypeer=false"
```

Comment:
same

Reply:
this one can probably be altered as the play url is only called once and it has options to the settings.

Reply:
use the verify_cert setting to append the verifypeer=false is needed